keyboards/handwired/ms_sculpt_mobile/handwired/rules.mk

```diff
@@ -0,0 +1,37 @@
+ifdef ASTAR
```

Comment:
I would prefer you split this up into separate `astar` and `teensypp` revisions, to avoid these ifdefs.